canonical-websites/www.ubuntu.com
# Changed background to white with logos on /server

## Done

* Changed background to white with logos on /server

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [server overview](http://0.0.0.0:8001/server)
- See that 'Built for big data and cloud' section is on a white background

## Issue / Card

Fixes #2345
